package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "xkcd-password",
-  "version": "1.2.0",
+  "version": "2.0.0",
```

Comment:
Ok sure that's exactly what I did but I guess the tag does not come through with the pull request.